snmp/metadata.csv

```diff
@@ -539,6 +557,14 @@ snmp.fwDropped,count,,packet,,[Checkpoint Firewall] Number of dropped packets.,0
 snmp.fwNumConn,gauge,,connection,,[Checkpoint Firewall] Number of concurrent IPv6 and IPv4 connections.,0,snmp,,
 snmp.fwPeakNumConn,gauge,,connection,,[Checkpoint Firewall] Peak number of concurrent connections since last reboot.,0,snmp,,
 snmp.fwRejected,count,,packet,,[Checkpoint Firewall] Number of rejected packets.,0,snmp,,
+snmp.haAPHbtTunnels,gauge,,,,"[WLSX-HA-MIB] The number of AP hbt GRE tunnels.",0,snmp,,
+snmp.haActiveAPs,gauge,,,,"[WLSX-HA-MIB] The number of active APs.",0,snmp,,
+snmp.haActiveVapTunnels,gauge,,,,"[WLSX-HA-MIB] The number of active per VAP GRE tunnels.",0,snmp,,
+snmp.haStandbyAPs,gauge,,,,"[WLSX-HA-MIB] The number of standby APs.",0,snmp,,
+snmp.haStandbyVapTunnels,gauge,,,,"[WLSX-HA-MIB] The number of standby per VAP GRE tunnels.",0,snmp,,
+snmp.haTotalAPs,gauge,,,,"[WLSX-HA-MIB] The total number of APs.",0,snmp,,
+snmp.haTotalVapTunnels,gauge,,,,"[WLSX-HA-MIB] The total number of per VAP GRE tunnels.",0,snmp,,
+snmp.healthDeviceTemperatureChas1MinAvg,gauge,,,,"[ALCATEL-IND1-HEALTH-MIB] Average chassis temperature over the last minute (percent).",0,snmp,,
```

Comment:
```suggestion
```

Is `healthDeviceTemperatureChas1MinAvg` intended? Maybe a error during merge conflicts?

Reply:
yes indeed it came from a merge conflict resolution

Reply:
@NouhaManai96 Could you revert/remove the change?

Reply:
done